.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,26 @@
+name: Tests E2E
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - 'master'
+jobs:
+  tests:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
```

Comment:
Why only ubuntu here? Wouldn't you want to run this on all major OS?

Reply:
macos doesn't work yet because of this - https://github.com/near/nearcore/issues/7235